parser/src/test/java/com/facebook/coresql/parser/TestSqlParser.java

```diff
@@ -66,4 +69,11 @@ public void parseUnparseTest()
             assertEquals(sql.trim(), unparse(ast).trim());
         }
     }
+
+    @Test
+    public void sqllogictest()
+            throws IOException
+    {
+        SqlLogicTest.run();
```

Comment:
Why is this a static method? If we want to run in parallel, we need different instances?

Reply:
Made non-static.

Done